sotodlib/tod_ops/filters.py

```diff
@@ -364,7 +365,8 @@ def timeconst_filter(target, freqs, tod, timeconst=None, invert=False):
     if timeconst is None:
         timeconst = 'timeconst'
     if isinstance(timeconst, str):
-        timeconst = tod[timeconst]
+        _f = attrgetter(timeconst)
+        timeconst = _f(tod)
```

Comment:
> 1. Why is this needed?  if tod is an axismanager, `tod[timeconst]` is actually better than `tod.<timeconst>`

Because time constant is buried multiple layers deep inside the axis manager it's in `tod['det_cal']['tau_eff']` passing `det_cal.tau_eff` to `attrgetter` can get things 2 layers deep like this.

> 2. Normally one would just do `timeconst = getattr(tod, timeconst)` instead of these two new lines

Just because I haven't use that/don't know how it works and we had an example of `attrgetter` for this already so I copied it. I'll try it out with `getattr` instead.

Reply:
Ah, neat.  Indeed attrgetter can do buried things, and getattr can't.  So just a add a brief comment to explain the purpose of that construction here.

Reply:
Done.